src/api-routes/persons/{osuId}/jobs.js

```diff
@@ -45,6 +56,10 @@ const post = async (req, res) => {
       return errorBuilder(res, 409, 'A job with the specified job ID already exists.');
     }
 
+    if (changeReasonCode !== 'AAHIR' && !studentEmployeeInd) {
+      return errorBuilder(res, 400, ['AAHIR change reason code must be used to create graduate employee records.']);
+    }
+
```

Comment:
Should we also catch if `AAHIR` is being used to create student job?

Reply:
As discussed, we should reach out to the HR to clarify what `changeReasonCode` should be used for student creation and also validate them here.